### PR TITLE
[ROX-11951 + ROX-11953] : Add csv export endpoints for image CVE and cluster CVE data

### PR DESCRIPTION
**Pull request by charmik-redhat**
## Description

- Added new routes for ImageCVE and ClusterCVE csv export.
- Added new handlers

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [x] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

Manual testing to check if new endpoints work correctly

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: central/main.go [hunk 1]
```diff
@@ -683,10 +685,22 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
...
 		customRoutes = append(customRoutes, routes.CustomRoute{
 			Route:         "/api/export/csv/node/cve",
-			Authorizer:    user.With(permissions.View(resources.Image), permissions.View(resources.Deployment), permissions.View(resources.Node)),
+			Authorizer:    user.With(permissions.View(resources.Node)),
 			ServerHandler: nodeCveCsv.NodeCVECSVHandler(),
 			Compression:   true,
 		})
+		customRoutes = append(customRoutes, routes.CustomRoute{
+			Route:         "/api/export/csv/image/cve",
+			Authorizer:    user.With(permissions.View(resources.Image), permissions.View(resources.Deployment)),
```
Comment (charmik-redhat) [inline central/main.go:694]: @md2119 Are the permissions sufficient/correct for each new endpoint?

Comment (md2119) [reply central/main.go:694]: No. You need to add deployment perm for image cves since the report is pulling deployment data as well. The rest looks good.

**Review by md2119 — APPROVED** [review]
address the comments and :shipit:

File: central/main.go (hunk 1 above)
Comment (charmik-redhat) [reply central/main.go:694]: @md2119 Deployment perm is added for image cves.

Comment (md2119) [reply central/main.go:694]: Ah, not sure how I missed that!

**Pull request merged**